tests/framework-tests/preferences/js/PrefsEditorTests.js

```diff
@@ -109,8 +126,8 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
             prefsEditor("#ui-options");
         };
 
-        var resetSaveCalled = function () {
-            saveCalled = false;
+        var resetSaveCalled = function (that) {
+            that.options.saveCalled = false;
```

Comment:
Must not write to a component's options - this was already corrected in my branch to the following:

https://github.com/amb26/infusion/blob/FLUID-5249/tests/framework-tests/preferences/js/PrefsEditorTests.js#L90